<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPointDifferenceCommandClass</class>
 <widget class="QDialog" name="AddPointDifferenceCommandClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddPointDifferenceCommand</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Добавить команду: Point Report</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>49</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Точка:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pointInput">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>391</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Допуск:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="thresholdInput">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>391</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>451</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
